Gulpfile.js

```diff
@@ -1,52 +1,45 @@
-const _MDLINT_    = `node ${RootFolder}/node_modules/markdownlint-cli/markdownlint.js`
+const fs            = require("fs");
+const path          = require("path");
+const _             = require("lodash");
+const gulp          = require("gulp");
+const {exec, spawn} = require("child_process");
+
+const RootFolder  = path.join();
+const DistFolder  = path.join(RootFolder, "Dist");
+const SrcFolder   = path.join(RootFolder, "Src");
+const TestFolder  = path.join(DistFolder, "Test");
+const NodeBin     = path.join(RootFolder, "node_modules", ".bin");
+
+const _AVA_       = path.join(NodeBin, "ava");
+const _C8_        = path.join(NodeBin, "c8") + " --reporter=lcov --reporter=html --reporter=text-summary";
+const _TSC_       = path.join(NodeBin, "ttsc");
+const _ESLINT_    = path.join(NodeBin, "eslint");
+const _MDLINT_    = path.join(NodeBin, "markdownlint");
 
 // ---------------------------------------------------------------------------------------------------------------------
-const DistPath = (aPath = "") => { return DistFolder + aPath; }
-const RootPath = (aPath = "") => { return RootFolder + aPath; }
-const SrcPath = (aPath = "") => { return SrcFolder + aPath; }
+const DistPath = (aPath = "") => path.join(DistFolder, aPath);
+const RootPath = (aPath = "") => path.join(RootFolder, aPath);
+const SrcPath = (aPath = "") => path.join(SrcFolder, aPath);
 
 const DistDest = (aPath = "") => { return gulp.dest(DistPath(aPath)); }
 const Root = (aPath = "") => { return gulp.src(RootPath(aPath)); }
 const Src = (aPath = "") => { return gulp.src(SrcPath(aPath)); }
 
 // ---------------------------------------------------------------------------------------------------------------------
-gulp.task("clean", done =>
-    {
-        del([DistPath("**/*")], { force: true });
-        done();
-    },
-);
+gulp.task("clean", done => spawnTask(_TSC_, done, ["--build", "--clean"]));
 
 // ---------------------------------------------------------------------------------------------------------------------
-gulp.task("compile", (done) => execTask(_TSC_, done));
+gulp.task("compile", (done) => spawnTask(_TSC_, done, ["--build"]));
 
 // ---------------------------------------------------------------------------------------------------------------------
 gulp.task("copy", gulp.parallel(
-    () => Root("/tsconfig.json").pipe(DistDest()),
-
-    () => Src("/Config/**/*").pipe(DistDest("/Src/Config")),
-    () => Src("/Config/**/.*").pipe(DistDest("/Src/Config")),
+    () => Root("tsconfig.json").pipe(DistDest()),
+    () => Root(path.join("Config", "**", "*")).pipe(DistDest("Config")),
+    () => Root(path.join("Config", "**", ".*")).pipe(DistDest("Config")),
 ));
 
 // ---------------------------------------------------------------------------------------------------------------------
-gulp.task("build", gulp.series("compile", "copy"));
+gulp.task("build", gulp.parallel("compile", "copy"));
```

Comment:
if we use the idea that Config doesn't hold any binary-relevant ts files, we can do these steps in parallel.